README.md

```diff
@@ -1,4 +1,4 @@
-# zkSync Era: The zkEVM LLVM Framework
+# zkSync Era: The EraVM LLVM Framework
```

Comment:
Every `zkEVM` that is not about the target itself should become `ZKsync` per our discussion on Slack!

Reply:
It should be ok now. thx!